runtime/doc/lua.txt

```diff
@@ -1718,6 +1718,15 @@ is_callable({f})                                           *vim.is_callable()*
     Return: ~
         (boolean) `true` if `f` is callable, else `false`
 
+is_truthy({v})                                               *vim.is_truthy()*
+    Determine truthy/falsy in vim script.
```

Comment:
To be clear, what I am extremely hesitant about is to add such a single-purpose convenience function to the global `vim` module. The name even suggests a very generic (Lua) function, which will add to confusion.